.github/workflows/setup-release-branch-protection.yml

```
@@ -0,0 +1,140 @@
+name: Setup Release Branch Protection
+
+on:
+  create:
+    branches:
```

actionlint (1.7.12)

[error] "branches" filter is not available for create event. it is only for merge_group, push, pull_request, pull_request_target, workflow_run events

(events)